sympy/solvers/ode/riccati.py

```diff
@@ -848,8 +862,11 @@ def solve_riccati(fx, x, b0, b1, b2, gensol=False):
         c.append(d)
         choices = product(*c)
         for choice in choices:
-            m, ybar = compute_m_ybar(x, poles, choice, -val_inf//2)
-            numy, deny = [Poly(e, x, extension=True) for e in ybar.together().as_numer_denom()]
+            m, ybar = compute_m_ybar(x, poles, choice, -val_inf // 2)
```

Comment:
One could add this operator to the config as well:
https://github.com/sympy/sympy/blob/2d90dc1d91b1d16679bec2c0f94d49b836b5315b/setup.cfg#L97

```cfg
no_spaces_around_selected_binary_operators = *,/,//
```

Reply:
f4c024c61a